decidim-dev/lib/generators/decidim/dummy_generator.rb

```diff
@@ -48,11 +48,6 @@ def create_dummy_app
       end
 
       def decidim_dev
-        # TODO: Remove these after PhantomJS updates WebKit version (see YML and
```

Comment:
Shouldn't we remove `autoprefixer` as a dependency as well then? It was mostly to prevent `PhantomJS` from breaking.

Reply:
Mmmm, I think we also need it so we can avoid vendor prefixes in our stylesheets... At least I haven't seen those `webkit-` / `moz-` / `ms-` prefixes anywhere.

Reply:
Ok, let's leave it like it is. Maybe one day we'll end up using Webpack. A man can dream...

Reply:
Yeah, that'd be awesome...